src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/06 PeriodEnd.PaymentsDue.Populate.Reference.RequiredPaymentsHistory.dml.sql

```diff
@@ -33,4 +33,5 @@ INSERT INTO [Reference].[RequiredPaymentsHistory]
 		UseLevyBalance
```

Comment:
consider replacing the where clause/sub query with an inner join on reference.providers. the current execution plan is a left semi join which is less performant than an inner join

 ```
SELECT
        Id,
        CommitmentId,
		CommitmentVersionId ,
		AccountId,
		AccountVersionId,
        LearnRefNumber,
		Uln,
        AimSeqNumber,
        rp.Ukprn,
        DeliveryMonth,
        DeliveryYear,
        CollectionPeriodName,
        CollectionPeriodMonth,
        CollectionPeriodYear,
        TransactionType,
        AmountDue,
		StandardCode,
		ProgrammeType,
		FrameworkCode,
		PathwayCode,
		PriceEpisodeIdentifier,
		LearnAimRef,
		LearningStartDate,
		rp.IlrSubmissionDateTime,
		ApprenticeshipContractType,
		SfaContributionPercentage,
		FundingLineType,
		UseLevyBalance
    FROM ${DAS_PeriodEnd.FQ}.PaymentsDue.RequiredPayments rp
	INNER JOIN Reference.Providers p ON rp.Ukprn = p.Ukprn
```